tests/unit_tests/gui/simulation/test_run_dialog.py

```diff
@@ -35,12 +36,12 @@ def test_success(runmodel, qtbot, mock_tracker):
 
     with qtbot.waitExposed(widget, timeout=30000):
         qtbot.waitUntil(lambda: widget._total_progress_bar.value() == 100)
-        assert widget.done_button.isVisible()
+        qtbot.waitUntil(widget.done_button.isVisible, timeout=100)
```

Comment:
Are you sure this isn' flaky? Couldn't there be some time between the total_progress_bar goes to 100 and the done button is visible?

Reply:
i think it's supposed to be instant - that's why there was no timeout.
but on inspecting the code a bit more carefully that is responsible for the done button, i don't really see in the immediate vicinity anything about progress being 100 - so it's at least not immediately clear.

do you suggest increasing the timeout i added?